docs/wiki/[User-Guide]-Deploy-Management-Resources.md

```diff
@@ -1 +1,141 @@
-_coming soon_
\ No newline at end of file
+## Overview
+This example will deploy resources for centrally managing the Enterprise Scale Landing Zone. This module will deploy the following resources with default configurations that can be adjusted once you are up and going:
+
+
+   * resource group
```

Comment:
Should we put this as a table which includes the default naming pattern and interpolation and also the resource group its deployed to etc.?